Gemfile

```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "~> 3.2.0"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0"
+gem "rails", "~> 7.0.7"
```

Comment:
Could we let this float and just `bundle update` to bump the lock file? That way we don't need to update this for every Rails update.

Reply:
I'm agreeing with you. But it's better if we could mention the exact versions for major dependencies in the gemfile.